repository/user_repository.py

```diff
@@ -21,4 +21,17 @@ def delete_user(self, email: str) -> None:
         user = self.get_user_by_email(email)
         if user:
             self.db.delete(user)
-            self.db.commit()
\ No newline at end of file
+            self.db.commit()
+    
+    def assign_mentor(self, mentor_email: str, mentee_email: str) -> None:
+        mentee = self.get_user_by_email(mentee_email)
+        mentor = self.get_user_by_email(mentor_email)
+        if mentee and mentor:
+            mentee.mentor_email = mentor_email  
```

Comment:
we should not have business logic inside the repository. We should only get the object we need from the db. All processing should be done in the service.